docs/source/examples/beam.py

```diff
@@ -51,7 +51,8 @@ def setup(self, N=4):
         return w[-1], [shear_eq, moment_eq, theta_eq, displ_eq, L == (N-1)*dx]
 
 
-b = Beam(N=10, substitutions={"L": 6, "EI": 1.1e4, "q": 110*np.ones(10)})
+b = Beam(N=6, substitutions={"L": 6, "EI": 1.1e4, "q": 110*np.ones(10)})
```

Comment:
I just lowered it as much as I could while still getting the right accuracy so that it'd be more sensitive to future changes.